lwbt/bte.c

```diff
@@ -555,73 +700,179 @@ s32 bte_registerdeviceasync(struct bte_pcb *pcb,struct bd_addr *bdaddr,s32 (*con
 	if(err!=ERR_OK) {
 		l2cap_close(l2capcb);
 		err = ERR_CONN;
-	}
+	}*/
 
 error:
 	_CPU_ISR_Restore(level);
-	//printf("bte_registerdeviceasync(%02x)\n",err);
+	printf("bte_registerdeviceasync(%02x)\n",err);
```

Comment:
Please use the LOG macro for logging

Reply:
a good thing to keep in mind for the final result. @Zarithya has to still clean it all up and test the temp syncing ^^
she has been doing this for a few weeks on her techy thuesday streams ^^